api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/AddAuthorsToInformationResourceGenerator.java

```diff
@@ -436,7 +496,7 @@ public void addFormSpecificData(EditConfigurationVTwo editConfiguration, VitroRe
 	    + "	 bind ( COALESCE(?firstName, \"\") As ?firstName1) . \n"
 	    + "	 bind ( COALESCE(?middleName, \"\") As ?middleName1) . \n"
 	    + "	 bind ( COALESCE(?lastName, \"\") As ?lastName1) . \n"
-	    + "	 bind (concat(str(?lastName1 + \", \"),str(?middleName1 + \" \"),str(?firstName1)) as ?authorName) . \n"
+	    + "	 bind (concat(str(str(?lastName1) + \", \"),str(str(?middleName1) + \" \"),str(?firstName1)) as ?authorName) . \n"
```

Comment:
Are the nested str method calls necessary here?

Reply:
The added str method calls are indeed necessary.